### PR TITLE
Update defaults

### PR DESCRIPTION
**Pull request by falkben**
This currently only changes max.depth (from `ceiling(log2(nrow(X)))`) to `0` and isn't quite ready to be merged

Other defaults we want to change are:

- [x] no min parent
- [x] stratify TRUE

from Slack and also #40

### CODE REVIEW

**Comment by MrAE** [comment]
Should max.depth be 0 or inf?

**Comment by falkben** [comment]
0 means that max.depth is not used which is what we want

**Comment by MrAE** [comment]
@falkben when you listed `no min.parent` that's equivalent to setting the default `min.parent = 1L` correct?

@jbrowne6

File: README.Rmd [hunk 1]
```diff
@@ -128,7 +128,7 @@ To create a forest the minimum data needed is an n by d input matrix (X) and an
 ```{r create-a-forest}
 X <- as.matrix(iris[,1:4])
 Y <- iris[[5L]]
-forest <- RerF(X, Y, seed = 1L, num.cores = 1L)
+forest <- RerF(X, Y, seed = 1L, num.cores = 1L, max.depth = ceiling(log2(nrow(X))))
```
Comment (falkben) [inline README.Rmd]: Do we care that outputs have changed in the readme.md (because defaults changed?) 

If readme outputs change, and assuming it doesn't lengthen run times at all, I am now thinking it would be better to undo the changes I made in this file setting the max.depth.  I don't think we have a desire to encourage people to modify these defaults to these (old) default values.

Comment (MrAE) [reply README.Rmd]: Agreed, I've removed the `max.depth` arguments
